pkg/executor/nginx/nginx.go

```diff
@@ -17,7 +17,7 @@ import (
 const nginxConfigTemplate string = `
 worker_processes  {{.WorkerProcesses}};
 error_log stderr;
-
+{{if .LogToFile}}error_log /u01/logs/nginx.log;{{end}}
```

Comment:
Siden du innfører denne, har du ikke lyst til å ha {{if !.LogToFile}}... på linjen over? Evt en if/else-konstruksjon?

Reply:
Access loggene har alltid blitt sendt til stdout. og jeg anser det ikke som en del av oppgaven å endre det.

Reply:
Problemet er at jeg _tror_ dette vil opphøre fordi `error_log /u01/logs/nginx.log;` overstyrer den konfigurasjonen. Oppgaven berører eksisterende funksjonalitet, mener jeg, og det er en del av oppgaven å oppklare endringen enten i kode eller doc.

Config som ender opp med å se slik ut, ser ikke riktig ut i mine øyne:
```
...
error_log stderr;
error_log /u01/logs/nginx.log;
...
```

Reply:
I det tilfellet du skisserer vil loggene havne begge steder.

Reply:
👍

Reply:
OK. Kan vurdere å være ekspliitt på dette i doc: "If `NGINX_INDEX_STRATEGY` is set to `file` logs are written to `/u01/logs/nginx.log` and `/u01/logs/nginx.access` in addition to stdout  /stderr"

Reply:
Enig. Legger til endringen